storage/s3/src/test/java/io/aiven/kafka/tieredstorage/storage/s3/S3StorageConfigTest.java

```diff
@@ -16,153 +16,133 @@
 
 package io.aiven.kafka.tieredstorage.storage.s3;
 
-import java.net.URL;
+import java.net.URI;
 import java.util.Map;
 
 import org.apache.kafka.common.config.ConfigException;
 
-import com.amazonaws.auth.AWSCredentialsProvider;
-import com.amazonaws.auth.AWSStaticCredentialsProvider;
-import com.amazonaws.auth.EnvironmentVariableCredentialsProvider;
-import com.amazonaws.regions.Regions;
-import com.amazonaws.services.s3.AmazonS3;
 import org.junit.jupiter.api.Test;
+import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
+import software.amazon.awssdk.auth.credentials.EnvironmentVariableCredentialsProvider;
+import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.s3.S3Client;
 
 import static io.aiven.kafka.tieredstorage.storage.s3.S3StorageConfig.S3_MULTIPART_UPLOAD_PART_SIZE_DEFAULT;
-import static io.aiven.kafka.tieredstorage.storage.s3.S3StorageConfig.S3_REGION_DEFAULT;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 class S3StorageConfigTest {
+    private static final String BUCKET_NAME = "b1";
+    private static final Region TEST_REGION = Region.US_EAST_2;
+    private static final String MINIO_URL = "http://minio";
 
     // Test scenarios
     // - Minimal/Full configs
```

Comment:
I think the full config test wasn't here before neither is here now, so probably
```suggestion
    // - Minimal config
```

Reply:
Fixed